sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/src/autorest.md

```diff
@@ -0,0 +1,17 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/f09aacf4c6b63be416212cb182f6b31e8bc6d545/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json
+library-name: HDInsight
```

Comment:
To avoid any possible issues, I suggest we use `HDInsightContainers` as library name to disambiguate from `Azure.ResourceManager.HDInsight`

Reply:
updated it to HDInsightContainers